mobile/src/main/java/com/caverock/androidsvg/IntegerParser.java

```diff
@@ -0,0 +1,177 @@
+/*

```

Comment:
>but I tried to follow the pattern used for de.duenndns.ssl

This is an issue which is on our list of things to fix, going forward we need to avoid this pattern
>Hi, not sure what you mean.

These classes were not created by us, not in our namespace and not under the EPL.   If there is a fix they have not updated yet, we can fork the repo and create a lib to include until such a time that they can fix it and we can use a gradle dependency.